forwardingproxy.go

```diff
@@ -113,11 +108,10 @@ func (p *ForwardingProxy) Proxy(w io.Writer, r io.ReadCloser, msg *proto.Control
 }
 
 func (p *ForwardingProxy) ServeHTTP(w http.ResponseWriter, r *http.Request, rr io.ReadCloser) {
-	p.logger.Log("Incoming request host", r.Host)
+	p.logger.Log("Incoming request host", r.Host, r.Header)
```

Comment:
You should probably follow the recipe for logging as is done all over this repo. You need to pass the level.